src/epgsql.erl

```diff
@@ -268,3 +286,27 @@ sync_on_error(C, Error = {error, _}) ->
 
 sync_on_error(_C, R) ->
     R.
+
+-spec standby_status_update(connection(), lsn(), lsn()) -> ok | error_reply().
+%% @doc sends last flushed and applied WAL positions to the server in a standby status update message via given `Connection'
+standby_status_update(Connection, FlushedLSN, AppliedLSN) ->
+    gen_server:call(Connection, {standby_status_update, FlushedLSN, AppliedLSN}).
+
+-spec start_replication(connection(), string(), Callback, cb_state(), string(), string()) -> ok | error_reply() when
+    Callback :: module() | pid().
+%% @doc instructs Postgres server to start streaming WAL for logical replication
+%% where
+%% `Connection'      - connection in replication mode
+%% `ReplicationSlot' - the name of the replication slot to stream changes from
+%% `Callback'        - Callback module which should have the callback functions implemented for message processing.
+%%                      or a process which should be able to receive replication messages.
+%% `CbInitState'     - Callback Module's initial state
+%% `WALPosition'     - the WAL position XXX/XXX to begin streaming at.
+%%                      "0/0" to let the server determine the start point.
+%% `PluginOpts'      - optional options passed to the slot's logical decoding plugin.
+%%                      For example: "option_name1 'value1', option_name2 'value2'"
+%% returns `ok' otherwise `{error, Reason}'
+start_replication(Connection, ReplicationSlot, Callback, CbInitState, WALPosition, PluginOpts) ->
+    gen_server:call(Connection, {start_replication, ReplicationSlot, Callback, CbInitState, WALPosition, PluginOpts}).
+start_replication(Connection, ReplicationSlot, Callback, CbInitState, WALPosition) ->
+    gen_server:call(Connection, {start_replication, ReplicationSlot, Callback, CbInitState, WALPosition, []}).
```

Comment:
No, I mean
```
start_replication(Connection, ReplicationSlot, Callback, CbInitState, WALPosition, PluginOpts) ->
    gen_server:call(Connection, {start_replication, ReplicationSlot, Callback, CbInitState, WALPosition, PluginOpts}).
start_replication(Connection, ReplicationSlot, Callback, CbInitState, WALPosition) ->
    start_replication(Connection, ReplicationSlot, Callback, CbInitState, WALPosition, []).
```

Because I more often see this cascadic style of writing functions with optional arguments then your variant (with 2 gen_server:call's)